lib/css/components/_stacks-lists.less

```diff
@@ -0,0 +1,29 @@
+//
+//  STACK OVERFLOW
+//
+//  This CSS comes from Stacks, our CSS & Pattern library for rapidly building
+//  Stack Overflow. For documentation of all these classes and how to contribute,
+//  visit https://stackoverflow.design/
+//
+//  TABLE OF CONTENTS
+//  • HORIZONTAL LISTS
+//
+
+//  ============================================================================
+//  $   HORIZONTAL LISTS
+//  ----------------------------------------------------------------------------
+
+.s-list.s-list__horizontal {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+
+    > li:not(:last-child)::after {
```

Comment:
@aaronshekey Do you prefer `:not(:last-child)::after` or `:not(:first-child)::before` for the bullets? It'd affect text styling applied to the `li`s and also the wrapping behaviour when it goes to a second line